tests/test_wheel.py

```diff
@@ -70,19 +72,21 @@ def test_read_valid(example_wheel):
 
 def test_read_non_existent_wheel_file_name():
     """Raise an exception when wheel file doesn't exist."""
-    file_name = "/foo/bar/baz.whl"
+    file_name = str(os.path.abspath(pathlib.Path("/foo/bar/baz.whl")))
```

Comment:
In general, I try to use `pathlib` exclusively, since it offers a lot of the same functionality as `os.path`. For example, I believe this is equivalent:

```suggestion
    file_name = str(pathlib.Path("/foo/bar/baz.whl").resolve())
```

However, I *think* the `str` might not be necessary, because the f-string will take care of that.